### PR TITLE
Reexport key-expansion

### PR DESCRIPTION
**Pull request by barvirm**
Reexport key-expansion #1424 .
I hope it's simple as this.

### CODE REVIEW

File: axum-extra/Cargo.toml [hunk 1]
```diff
@@ -18,6 +18,7 @@ async-read-body = ["dep:tokio-util", "tokio-util?/io"]
 cookie = ["dep:cookie"]
 cookie-private = ["cookie", "cookie?/private"]
 cookie-signed = ["cookie", "cookie?/signed"]
+cookie-key-expansion = ["cookie", "cookie?/key-expansion"]
```
[the remark this answers is not in the document]
Comment (barvirm) [reply axum-extra/Cargo.toml:21]: I don't think it's necessary to enable key-expansion with signed and private cookies. Both of these can be used without this feature.

[the remark this answers is not in the document]
Comment (davidpdrsn) [reply axum-extra/Cargo.toml:21]: Alright fair enough. I don't feel strongly either way. 

Wanna add it to lib.rs as well?

[the remark this answers is not in the document]
Comment (barvirm) [reply axum-extra/Cargo.toml:21]: I pushed the description into `lib.rs`.

**Review by davidpdrsn — APPROVED** [review]
Thanks!

**Pull request merged**